lib/Dialect/TritonNvidiaGPU/IR/Dialect.cpp

```diff
@@ -254,10 +254,11 @@ LogicalResult TensorMemoryEncodingAttr::verify(
     return emitError() << "CTASplitM and CTASplitN must be greater than 0";
   }
   if (blockM != 64 && blockM != 128) {
-    return emitError() << "blockM must be 64 or 128";
+    return emitError() << "blockM must be 64 or 128 but got " << blockM;
   }
   if (!llvm::isPowerOf2_32(blockN) || blockN > 512) {
```

Comment:
Unrelated, but I just realised that this should be blockN < 512 * (isUnpacked ? (32 / bitwidth) : 1). I can add it to another PR if you don't want to fix it in this one tho.